src/views/Settings.vue

```diff
@@ -13,7 +13,7 @@
           <ion-icon :icon="storefrontOutline" slot="start" />
           <ion-label>{{$t("store")}}</ion-label>
           <ion-select interface="popover" :placeholder="$t('store name')" :selected-text="currentFacility.facilityId" @ionChange="setFacility($event)">
-            <ion-select-option v-for="facility in userProfile.facilities" :key="facility.facilityId" :value="facility.facilityId" >{{ facility.facilityId }}</ion-select-option>
+            <ion-select-option v-for="facility in userProfile?.facilities" :key="facility.facilityId" :value="facility.facilityId" >{{ facility.facilityId }}</ion-select-option>
```

Comment:
I think this will also break, as here we are checking that the `userProfile` should not be null or undefined we are not checking for facilities being null or undefined.
Please refer
https://github.com/hotwax/bopis/blob/main/src/views/Settings.vue#L16